uswid/__init__.py

```
@@ -2,9 +2,22 @@
 
+try:
+    from ._version import version as __version__
+except (ImportError, AttributeError):
+    __version__ = "0.0.0+unknown"
+else:
+    if __version__ == "0.0.0":
+        try:
+            from importlib.metadata import version as _meta_version
+
+            __version__ = _meta_version("uswid")
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)